src/DaemonExec.ts

```diff
@@ -0,0 +1,244 @@
+import fs = require('fs')
+//import path = require('path')
+import Docker = require('dockerode')
+//needs docker to already be running
+const docker = new Docker()
+
+const listImages = async () => {
+	try {
+		const images = await docker.listImages({})
+		return images
+	} catch (err) {
+		console.log(err)
+		return []
+	}
+}
+
+const listContainers = async () => {
+	try {
+		const containers = await docker.listContainers()
+		return containers
+	} catch (err) {
+		console.log(err)
```

Comment:
Since this is a library, it shouldn't print anything to console (same applies to the other functions like this).